src/test/ui/nll/closures-in-loops.stderr

```diff
@@ -13,17 +13,21 @@ error[E0499]: cannot borrow `x` as mutable more than once at a time
   --> $DIR/closures-in-loops.rs:13:16
    |
 LL |         v.push(|| x = String::new());
-   |                ^^ - borrows occur due to use of `x` in closure
-   |                |
-   |                `x` was mutably borrowed here in the previous iteration of the loop
+   |         -------^^-------------------
+   |         |      |  |
+   |         |      |  borrows occur due to use of `x` in closure
+   |         |      `x` was mutably borrowed here in the previous iteration of the loop
+   |         that borrow lives into the next loop iteration, preventing another mutable borrow because the borrow later used here
```

Comment:
This isn't gramatically correct, it should be `preventing another mutable borrow because that borrow is later used here`

Reply:
Corrected.